modules/operator/kubeconfig.tf

```diff
@@ -20,18 +20,18 @@ resource null_resource "get_kubeconfig" {
     bastion_private_key = file(var.olcne_bastion.ssh_private_key_path)
   }
 
-  depends_on = [null_resource.install_module]
+  depends_on = [null_resource.install_kubernetes_module]
```

Comment:
Neat. In 1.0. we had only kubernetes as module I think, now we need a better way of distinguishing the modules and use the supplied one if possible.